tests/helpers/utils.py

```diff
@@ -28,6 +28,7 @@
 import pytest
 
 from qutebrowser.utils import qtutils
+from PyQt5.QtCore import QObject, pyqtSignal
```

Comment:
Please move this in a block before the `qutebrowser` import (see [pep8](https://www.python.org/dev/peps/pep-0008/#imports)).